src/device/rdk/interface.rs

```diff
@@ -353,38 +353,50 @@ pub fn service_is_available(service: &str) -> Result<bool, DabError> {
 lazy_static! {
     static ref RDK_KEYMAP: HashMap<String, u16> = {
         let mut keycode_map = HashMap::new();
-        keycode_map.insert(String::from("KEY_POWER"),116);
-        keycode_map.insert(String::from("KEY_HOME"),36);
-        keycode_map.insert(String::from("KEY_VOLUME_UP"),175);
-        keycode_map.insert(String::from("KEY_VOLUME_DOWN"),174);
-        keycode_map.insert(String::from("KEY_MUTE"),173);
-        // keycode_map.insert(String::from("KEY_EXIT"),27);
-        keycode_map.insert(String::from("KEY_UP"),38);
-        keycode_map.insert(String::from("KEY_PAGE_UP"),33);
-        keycode_map.insert(String::from("KEY_PAGE_DOWN"),34);
-        keycode_map.insert(String::from("KEY_RIGHT"),39);
-        keycode_map.insert(String::from("KEY_DOWN"),40);
-        keycode_map.insert(String::from("KEY_LEFT"),37);
-        keycode_map.insert(String::from("KEY_ENTER"),13);
-        keycode_map.insert(String::from("KEY_BACK"),8);
-        keycode_map.insert(String::from("KEY_PLAY"),13);
-        keycode_map.insert(String::from("KEY_PLAY_PAUSE"),227);
-        keycode_map.insert(String::from("KEY_PAUSE"),19);
-        // keycode_map.insert(String::from("KEY_STOP"),178);
-        keycode_map.insert(String::from("KEY_REWIND"),224);
-        keycode_map.insert(String::from("KEY_FAST_FORWARD"),223);
-        keycode_map.insert(String::from("KEY_SKIP_REWIND"),34);
-        keycode_map.insert(String::from("KEY_SKIP_FAST_FORWARD"),33);
-        keycode_map.insert(String::from("KEY_0"),48);
-        keycode_map.insert(String::from("KEY_1"),49);
-        keycode_map.insert(String::from("KEY_2"),50);
-        keycode_map.insert(String::from("KEY_3"),51);
-        keycode_map.insert(String::from("KEY_4"),52);
-        keycode_map.insert(String::from("KEY_5"),53);
-        keycode_map.insert(String::from("KEY_6"),54);
-        keycode_map.insert(String::from("KEY_7"),55);
-        keycode_map.insert(String::from("KEY_8"),56);
-        keycode_map.insert(String::from("KEY_9"),57);
+
+        if let Ok(json_file) = read_platform_config_json("/etc/dab/keymap.json") {
```

Comment:
Could you please add a note: expected format of the file contents.
Ref: https://github.com/device-automation-bus/dab-adapter-rs/pull/74/files#diff-5069a34dcc4ad54a16c610515cb8b8ee142edcc85b9e46dd37d26fe0c9c2a130R403